Resources/views/Collector/migrations.html.twig

```diff
@@ -65,32 +84,91 @@
 {% endblock %}
 
 {% block panel %}
+    {% set num_executed_migrations = collector.data.executed_migrations|length %}
+    {% set num_unavailable_migrations = collector.data.unavailable_migrations_count %}
+    {% set num_available_migrations = collector.data.available_migrations_count %}
+    {% set num_new_migrations = collector.data.new_migrations|length %}
+
     <h2>Doctrine Migrations</h2>
     <div class="metrics">
         <div class="metric">
-            <span class="value">{{ collector.data.executed_migrations|length }}</span>
+            <span class="value">{{ num_executed_migrations }}</span>
             <span class="label">Executed</span>
         </div>
+
+        {% if profiler_markup_version >= 3 %}
+            <div class="metric-group">
+        {% endif %}
+
         <div class="metric">
-            <span class="value">{{ collector.data.unavailable_migrations_count }}</span>
-            <span class="label">Executed Unavailable</span>
+            <span class="value">{{ num_unavailable_migrations }}</span>
+            <span class="label">Unavailable</span>
         </div>
         <div class="metric">
-            <span class="value">{{ collector.data.available_migrations_count }}</span>
+            <span class="value">{{ num_available_migrations }}</span>
             <span class="label">Available</span>
         </div>
         <div class="metric">
-            <span class="value">{{ collector.data.new_migrations|length }}</span>
+            <span class="value">{{ num_new_migrations }}</span>
             <span class="label">New</span>
         </div>
+
+        {% if profiler_markup_version >= 3 %}
+            </div> {# closes the <div class="metric-group"> #}
+        {% endif %}
```

Comment:
@javiereguiluz long time not working with twig, but I think a better way to do this might be:

```html+jinja
{% set metric_group %}
<!-- html goes here -->
{% endset %}
{% if profiler_markup_version >= 3 %}
    <div class="metric-group">{{ metric_group }}</div>
{% else %}
    {{metric_group}}
{% endif %}
```